### PR TITLE
API Remove GraphQL

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1799

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,5 +4,3 @@
 /themes/simple/
 /_resources/
 /public/_resources/
-/.graphql-generated/
-/public/_graphql/
```
Comment (ChloeSartorelli) [inline .gitignore:8]: We should probably keep these - people using public schemas shouldn't suddenly find their new projects are adding these to their git repos.

Alternatively we should make absolutely sure this is called out in the graphql docs early on.

File: composer.json [hunk 2]
```diff
@@ -16,8 +16,7 @@
     "extra": {
         "project-files": [
             "app/_config/*",
-            ".env.example",
-            ".graphql-generated/*"
+            ".env.example"
```
Comment (ChloeSartorelli) [inline composer.json]: We should either keep this (along with the folder, which I note you haven't deleted) - or else make sure the manifest exclude file gets generated by silverstripe/graphql itself.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (emteknetnz) [reply .gitignore:8]: Reverted

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json]: Reverted

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge when other PRs are happy

**Pull request merged**